test/functional/testlib.bash

```diff
@@ -1754,14 +1755,14 @@ create_bundle() { # swupd_function
 		# this file is added in every bundle by default, it would add too much overhead
 		# for most tests
 		fdir=$(dirname "${val%:*}")
-		if [ ! "$(sudo cat "$manifest" | grep -x "D\\.\\.\\..*$fdir")" ] && [ "$fdir" != "/usr/share/clear/bundles" ] \
+		if [ ! "$(sudo cat "$manifest" | grep -x "[D|L]\\.\\.\\..*$fdir")" ] && [ "$fdir" != "/usr/share/clear/bundles" ] \
```

Comment:
Should be `[DL]` instead of `[D|L]`, correct? If you did need alternation (in this case, it's not needed), then `-E` would be required option to grep, and the alternation would be written as `(D|L)` instead.

Reply:
I'll fix that, thanks